### ISSUE TITLE
WCAG Failure: Color contrast of thumbnail text on thumbnail is too low

### ISSUE THREAD

**Issue by patdunlavey** [ProjectMirador/mirador#3917]
From an accessibility audit of our Mirador 3 install:
- Failed WCAG Success Criteria: 1.4.3 Contrast (Minimum) (AA)
- Color contrast of thumbnail text on thumbnail is too low at 4:1 vs. 4.5:1 required.

Recommendation: Use a darker gray on the thumbnail text.

**Comment by lhenze** [comment]
Hi there, looking at this in the context of Mirador 4 --  what I see is white text on top of each image now.  Am I looking in the right place?  If so, yes, this is still an issue, because the background of the text is transparent, and so the contrast can't be predicted.  
![Image](https://github.com/user-attachments/assets/ae75f6aa-cb00-4494-8ee5-8c8246dfd82f)

**Comment by lhenze** [comment]
To amend this a bit, there is a now gradient behind the text.  If I put an all-white image in the thumbnail, the pixels for the background read as #AAAAAA at the top, which set behind white text provides a 2.32 contrast. So the gradient helps a lot but it's still potentially below the required contrast.  

![Image](https://github.com/user-attachments/assets/7516e6a1-9f63-4464-98ac-de379ff75524)

Here's what it would look like with gradient removed at background set at #757575, providing 4.61 contrast. 

![Image](https://github.com/user-attachments/assets/b58e6bbe-0044-45f6-ba76-b7986dc45504)

Thoughts on this, other design ideas?

**Comment by lhenze** [comment]
We've discussed -- it seems hard to have a solution that will serve every use case right now.
Before making decisions about the default display, can we explore adding an option for hiding the label entirely?   
We are looking at the settings 
https://github.com/ProjectMirador/mirador/blob/main/src/config/settings.js#L262 
Over to @dismorfo

**Comment by marlo-longley** [comment]
Audit 2026:

Developer Actions>
- Adopt text-under-thumbnail design from this thread
- Add config setting for hiding the label entirely

**Comment by gdicristofaro** [comment]
I believe this may be resolved in https://github.com/ProjectMirador/mirador/pull/4409
